src/features/Tiles/MovieTiles/index.js

```diff
@@ -12,55 +14,113 @@ import {
     Tags,
     Tag,
     ContentContainer,
+    ShowMoreButton
 } from "./styled";
 import { Rate } from "../../Rate";
+import { Arrow } from "../Arrow";
 
 const MovieTiles = ({ title }) => {
     const popularMoviesData = useSelector(selectPopularMoviesData);
-    const moviesList = popularMoviesData.results;
+
+    const imageURL = "http://image.tmdb.org/t/p/";
+    const size = "w300";
+    const poster = `${imageURL}${size}`;
+
+    let moviesList;
+
+    const [showMore, setShowMore] = useState(false);
     const moviesGenresData = useSelector(selectGenres);
     const moviesGenres = moviesGenresData.genres;
+    const sectionName = title.toLowerCase();
+    const creditsData = useSelector(selectPersonCredits);
+
+
+    switch (sectionName) {
+        case "cast":
+            moviesList = creditsData.cast;
+            break;
+
+        case "crew":
+            moviesList = creditsData.crew;
+            break;
+
+        default:
+            moviesList = popularMoviesData.results;
+    }
 
-    return (
+    return ( moviesList.length ?
         <>
-            <SiteTitle>{title}</SiteTitle>
-            <List>
-                {moviesList.map((movie, index) =>
-                    <li key={index}>
-                        <Tile
-                            to={`/movie/${movie.id}`}
-                        >
-                            <Picture
-                                src={movie.poster_path ? `https://image.tmdb.org/t/p/w300${movie.poster_path}`
-                                    : pictureSubstitution
-                                }
-                                alt={`${title} poster`}
-                            />
-                            <ContentContainer>
-                                <TileTitle>{movie.title}</TileTitle>
-                                {movie.release_date && (
-                                    <Year>
-                                        {movie.release_date.slice(0, 4)}
-                                    </Year>
-                                )}
+        
+            <section>
+                {
+                    sectionName.includes("popular")
+                        ?
+                        <SiteTitle>
+                            {sectionName.charAt(0).toUpperCase() + sectionName.slice(1)}
+                        </SiteTitle>
+                        :
+                        <SiteTitle>
+                            {`Movie - ${sectionName} (${moviesList.length})`}
+                        </SiteTitle>
+                }
+                <List>
+                    {moviesList.map((movie, index) =>
```

Comment:
please make destructuring:
 {moviesList.map(({
                        id,
                        poster_path,
                        title,
                        release_date,
                        character,
                        job,
                        genre_ids,
                        vote_average,
                        vote_count },
                        index) =>
                        
                        and remove all movies.[...] in map paths

Reply:
Done!